release/models/network-instance/openconfig-evpn.yang

```diff
@@ -1144,16 +1207,52 @@ module openconfig-evpn {
     }
   }
 
-  grouping evpn-endpoint-vni-state {
+  grouping evpn-endpoint-vni-config {
     description
-      "Grouping for L2VNI and L3VNI state information learned on the
-      local VXLAN Tunnel End Point from remote VTEPs";
+      "Grouping for L2VNI and L3VNI configuration parameters";
 
     leaf vni {
       type oc-evpn-types:evi-id;
       description "L2VNI or L3VNI Identifier";
     }
 
+    leaf vni-type {
+      type enumeration {
+        enum L2 {
+          description
+            "This is a Layer 2 service virtual network identifier (L2VNI)
+            that is used for communication within the same subnet or
+            broadcast domain";
+        }
+        enum L3 {
+          description
+            "This is a Layer 3 service virtual network identifier (L3VNI)
+            or VRF VNI that is used for communication between subnets";
+        }
+      }
+      description "The type of virtual network identfier";
+    }
+
+    leaf bridge-domain {
+      type uint32;
+      description
+        "This reflects the configured VLAN or Bridge Domain that maps to this
+        L2VNI in the VXLAN fabric";
+    }
```

Comment:
In OC, if the the network-instance is of type `L2VSI`, then that also indicates the bridge domain.  So I think this leaf is not needed.

E.g.
https://github.com/openconfig/public/blob/4d4aa4b7dd94acfae84e3f567b534116e6bd5ca4/release/models/network-instance/openconfig-network-instance-types.yang#L158-L162

Ref: https://github.com/openconfig/public/blob/master/doc/network_instance_examples.md

Reply:
Similarly to `l3-vrf-name`, this is intended to be symmetric with the existing `endpoint-vnis` container, for similar reasons.

Reply:
I see, ok.   I think vlan is likely a better name, as we don't use 'bridge' often in OC.  Wouldn't this leaf refer directly to an OC configured vlan. (ie: /interfaces/interface/subinterfaces/subinterface/vlan/config/vlan-id) ?

Reply:
In practice yes, I'm assuming the reason it's this way in the first place is to make the distinction between a VLAN as a bridging domain, and a VLAN as a specific dot1q tag (where multiple tags could refer to the same bridging domain, based on other config)

Reply:
Ok, resolving this as it's expanding on the state leaf already merged in to OC (https://github.com/openconfig/public/pull/437) and the desire to not create a breaking change over the choice of name for a leaf.